SDKBuildScripts/Update-PlayFabSdk.ps1

```diff
@@ -230,29 +230,29 @@ begin
 
 process
 {
-    foreach($sdkName in $SdkName)
+    foreach($targetSdkName in $SdkName)
```

Comment:
>$SdkName [](start = 30, length = 8)

Perhaps we could rename the $SdkName variable to $SdkNames to reflect it's a collection of string?

Reply:
I think I agree with @julioct.
I mildly prefer to rename SdkName to indicate plurality (SdkNames, SdkNameList, etc)

Reply:
I actually had it as $SdkNames before but I changed it to $SdkName because that seemed to be the standard that Powershell uses for most of it's APIs.

For example `Get-Item` has a `-Path` parameter which is a `string[]`, so you can either pass in one value or multiple values.  This prevents you from having to use `Update-PlayFabSdk -SdkNames CSharp ...` which can feel a little odd.

I'm happy to change it if you prefer, I just wanted to explain my reasoning first. :)

Reply:
Fair enough, no issues then. Maybe just add a private $sdkNames variable to use in this loop while keeping the original as the public parameter

---
In reply to: [296388986](https://github.com/PlayFab/SDKGenerator/pull/573#discussion_r296388986) [](ancestors = 296388986)

Reply:
Are you saying you just want to have something this?

```powershell
$sdkNames = $SdkName;
foreach($targetSdkName in $sdkNames)
{
  ...
}
```

Reply:
Yea, but not feeling super strong there. Not a big deal

---
In reply to: [296408628](https://github.com/PlayFab/SDKGenerator/pull/573#discussion_r296408628) [](ancestors = 296408628)

Reply:
Okay, gonna leave this as is for now.  @pgilmorepf, can you merge whenever?